cpp/README.rst

```diff
@@ -58,6 +58,16 @@ undergoing active development. Refer to
 [this GitHub issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/4133)
 for more information.
 
+Build steps
+-----------
+To build the docker image, run the following command from the shell. This command must be run in
+# the "aws-doc-sdk-examples" directory, the parent directory of "cpp", in order to access the resources folder.

```

Comment:
You must remove the # signs. In the dockerfile you copied this from, these were comments. In the ReadMe, they indicate header style.

Reply:
Pushed another version. Sorry!